runtime/src/iree/hal/utils/wrapped_buffer.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_HAL_UTILS_WRAPPED_BUFFER_H_
+#define IREE_HAL_UTILS_WRAPPED_BUFFER_H_
+
+#include "iree/base/api.h"
+#include "iree/hal/api.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+typedef struct iree_hal_wrapped_buffer_t iree_hal_wrapped_buffer_t;
+
+bool iree_hal_wrapped_buffer_isa(iree_hal_buffer_t *base_value);
```

Comment:
style here and elsewhere: asterisks stay with the type
```suggestion
bool iree_hal_wrapped_buffer_isa(iree_hal_buffer_t* base_value);
```